parts/kubernetesagentcustomdata.yml

```diff
@@ -123,6 +123,7 @@ write_files:
             --cluster-domain=cluster.local \
             --cloud-provider=azure \
             --cloud-config=/etc/kubernetes/azure.json \
+            --azure-container-registry-config=/etc/kubernetes/azure.json \
```

Comment:
This was pulled back out, we need other changes first, will do in a separate PR.